README.md

```diff
@@ -1,178 +1,176 @@
+[![Try TimescaleDB for free](https://img.shields.io/badge/Try_Timescale_for_free-black?style=for-the-badge&logo=timescale&logoColor=white)](https://console.cloud.timescale.com/signup)
 
-To determine which option is best for you, see [Timescale Products](https://tsdb.co/GitHubTimescaleProducts)
-for more information about our Apache-2 version, TimescaleDB Community (self-hosted), and Timescale
-Cloud (hosted), including: feature comparisons, FAQ, documentation, and support.
+</div>
 
-Below is an introduction to TimescaleDB. For more information, please check out
-these other resources:
-- [Developer Documentation](https://docs.timescale.com/getting-started/latest/services/)
-- [Slack Channel](https://slack-login.timescale.com)
-- [Timescale Community Forum](https://www.timescale.com/forum/)
-- [Timescale Release Notes & Future Plans](https://tsdb.co/GitHubTimescaleDocsReleaseNotes)
+TimescaleDB scales PostgreSQL for time-series data with the help of [hypertables](https://docs.timescale.com/use-timescale/latest/hypertables/about-hypertables/). Hypertables are PostgreSQL tables that automatically partition your data by time and space. You interact with a hypertable in the same way as regular PostgreSQL table. Behind the scenes, the database performs the work of setting up and maintaining the hypertable's partitions.
 
-For reference and clarity, all code files in this repository reference
-licensing in their header (either the Apache-2-open-source license
-or [Timescale License (TSL)](https://github.com/timescale/timescaledb/blob/main/tsl/LICENSE-TIMESCALE)
-). Apache-2 licensed binaries can be built by passing `-DAPACHE_ONLY=1` to `bootstrap`.
+From the perspective of both use and management, TimescaleDB looks and feels like PostgreSQL, and can be managed and queried as
+such. However, it provides a range of features and optimizations that make managing your time-series data easier and more efficient.
 
-[Contributors welcome.](https://github.com/timescale/timescaledb/blob/main/CONTRIBUTING.md)
+| **Get started with TimescaleDB** |**Learn more about TimescaleDB**|
+|---|--|
+| <ul><li>[Install](#install-timescaledb)</li><li>[Create a hypertable](#create-a-hypertable)</li><li>[Insert and query data](#insert-and-query-data)</li><li>[Compress data](#compress-data)</li><li>[Create time buckets](#create-time-buckets)</li><li>[Create continuous aggregates](#create-continuous-aggregates)</li><li>[Back up and replicate data](#back-up-replicate-and-restore-data)</li></ul> | <ul><li>[Developer documentation](https://docs.timescale.com/)</li><li>[Release notes](https://tsdb.co/GitHubTimescaleDocsReleaseNotes)</li><li>[Testing TimescaleDB](test/README.md)</li><li>[Timescale community forum](https://www.timescale.com/forum/)</li><li>[GitHub issues](https://github.com/timescale/timescaledb/issues)</li><li>[Timescale support](https://tsdb.co/GitHubTimescaleSupport)</li></ul>|
```

Comment:
```suggestion
## Get started with TimescaleDB
* [Install](#install-timescaledb)
* [Create a hypertable](#create-a-hypertable)
* [Insert and query data](#insert-and-query-data)
* [Compress data](#compress-data)
* Create time buckets](#create-time-buckets)
* [Create continuous aggregates](#create-continuous-aggregates)
* [Back up and replicate data](#back-up-replicate-and-restore-data)

## Learn more about TimescaleDB
* [Developer documentation](https://docs.timescale.com/)
* [Release notes](https://tsdb.co/GitHubTimescaleDocsReleaseNotes)
* [Testing TimescaleDB](test/README.md)
* [Timescale community forum](https://www.timescale.com/forum/)
* [Timescale support](https://tsdb.co/GitHubTimescaleSupport)
```

Reply:
Hi @philkra , the idea behind this table is that when people read the page they see the things they can do, and the first major action (Install), https://github.com/timescale/timescaledb/blob/bfe181d5444e9c4a1cf3fd3ce5ea54beccfcc01f/README.md (Yes, we need to sort out the links). If you put one list after another then you see two lists. Your call.

Reply:
Thanks @billy-the-fish for the context, the idea resonates with me. I'm on the fence, let's go with the table. We can adjust if necessary